sdk/storage/tests.yml

```diff
@@ -47,6 +50,19 @@ stages:
           Path: sdk/storage/platform-matrix.json
           Selection: sparse
           GenerateVMJobs: true
+      PostSteps:
+        - task: Maven@3
```

Comment:
Do we want to run this task if the test run fails? Also, if this step fails do we want to fail the job?

Reply:
I think we can live with this not running if tests fail. We want to have green build. I'll check the later.

Reply:
![image](https://user-images.githubusercontent.com/61715331/116643923-f3426c00-a926-11eb-895b-1bdefd17a135.png)